### PR TITLE
Oauth

### PR DESCRIPTION
**Pull request by jemyishai**
Connects to #24 

Facebook and google api endpoints for auth

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ node_modules
 public/bundle.js
 public/bundle.js.map
 secrets.js
+secrets.txt
```
Comment (jluria) [inline .gitignore:5]: 🐼

Comment (schai6) [reply .gitignore:5]: Should probably use secrets.js, or remove secrets.js.

File: client/components/auth-form.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ const AuthForm = (props) => {
         </div>
         {error && error.response && <div> {error.response.data} </div>}
       </form>
-      <a href="/auth/google">{displayName} with Google</a>
+      <a href="/auth/google">{displayName} with Google</a> <br />
```
Comment (schai6) [inline client/components/auth-form.js:28]: What is this `<br />` tag?

**Review by schai6 — CHANGES_REQUESTED** [review]
Looks good! Just a few things...

File: client/components/auth-form.js (hunk 2 above)
Comment (johnson-alan) [reply client/components/auth-form.js:28]: It splits the line so the two Oauth links aren't side by side.

Comment (johnson-alan) [reply client/components/auth-form.js:28]: Could also be wrapped in divs later (probably will be later)

**Pull request closed without merging**